source/StepBro.Core/Parser/FileBuilder.cs

```diff
@@ -875,6 +875,17 @@ public static int ParseFiles(ServiceManager services, ILogger logger, IScriptFil
                                 walker.Walk(listener, context);
                                 file.UpdateRootIdentifiers();
                             }
+                            catch (Exception e)
+                            {
+                                StringBuilder exceptionMessage = new StringBuilder();
+                                exceptionMessage.Append("File: ").Append(file.FileName).Append('.');
```

Comment:
No need for this anymore :-) 
file.ErrorsInternal belongs the the file being parsed.